kubernetes-pipeline/values.yaml

```diff
@@ -31,6 +31,9 @@ jenkins:
       ingress.kubernetes.io/ssl-redirect: 'true'
       kubernetes.io/ingress.class: nginx
       kubernetes.io/tls-acme: "true"
+  jobs:
+    baseImageUpdate:
+      cronFrequency: 'daily'
```

Comment:
Shall we rename this to just "cron"?. Also, the default value should be '@daily'. We can point this doc[1]https://www.jenkins.io/doc/book/pipeline/syntax/#cron-syntax for further details.

Reply:
+1